src/components/TextField/index.tsx

```diff
@@ -222,7 +222,7 @@ const TextField: FunctionComponent<TextFieldProps> = forwardRef(
     );
 
     return (
-      <WithLabel label={label} hintText={hintText} required={props.required || false}>
+      <WithLabel theme={theme} label={label} hintText={hintText} required={props.required || false}>
```

Comment:
This has broken theming in snapshots, please reinstate theme being passed into with label
![Screenshot 2021-02-16 at 14 01 28](https://user-images.githubusercontent.com/5347198/108073045-8590de80-705f-11eb-8c6b-1fbe86292c17.png)

Reply:
ah we actually fixed the snap, cool